src/exosite.c

```diff
@@ -376,8 +426,20 @@ exo_state exo_operate(exo_op *op, uint8_t count)
       break;
     }
   }
+}
+
+// process all ops that are in an active state
+static void exo_process_active_ops(exo_op *op, uint8_t count)
+{
+  uint8_t buf[576];
```

Comment:
Ya, that's probably the best option. But, for now I'm just going to leave it hard coded. I did change it to use a named const that I hope is descriptive.